tests/test_basicio.py

```diff
@@ -10,6 +10,8 @@
 
 TESTDATA = "apple\nbanana\norange\ngrape"
 TESTWRITEDATA = "the end is never the end is never the end"
+sleep_time = 1
+expiry_time = 0.1
```

Comment:
If expiry_time is 0, will the tests pass with sleep time of 0?

Reply:
One test fails, the one that specifically tests the dircache. That is to be expected since a nonzero expiry time is necessary for the dircache to be used